packages/commerce-server/src/format-prices-for-product.ts

```diff
@@ -181,15 +172,28 @@ export async function formatPricesForProduct(
     ? appliedMerchantCoupon.percentageDiscount.toNumber() < bulkCouponPercent
     : true
 
+  const hasNonPPPPurchases = userPurchases.some(
+    (purchase) => purchase.status === 'Valid',
+  )
+
   const pppConditionsMet =
-    quantity === 1 && pppDiscountPercent > 0 && appliedMerchantCouponLessThanPPP
+    pppDiscountPercent > 0 &&
+    quantity === 1 &&
+    !hasNonPPPPurchases &&
+    appliedMerchantCouponLessThanPPP
```

Comment:
if they've made any non-PPP purchases, they don't get PPP which seems fair, mixing and matching makes life hard and PPP is meant as a courtesy